Lombiq.HelpfulLibraries.LinqToDb/Docs/LinqToDb.md

```diff
@@ -0,0 +1,45 @@
+# Lombiq Linq2DB for Orchard Core
+LINQ to DB is the fastest LINQ database access library offering a simple, light, fast, and type-safe layer between your POCO objects and your database.
+Read more about this [library here](https://github.com/linq2db/linq2db).
+
+## Sample
+
+Enable `Lombiq.HelpfulLibraries.LinqToDb` feature and use dependency injection for the dataconnection.
+Keep in mind, that the LinqToDbConnection object is [disposable](https://linq2db.github.io/articles/general/Managing-data-connection.html)!
+```
+public class LinqToDbSamplesController
+    {
+        private readonly LinqToDbConnection _db;
+        public LinqToDbSamplesController(LinqToDbConnection db) => _db = db;
+
+        public async Task<ActionResult> SimpleQuery()
+        {
+            using (_db) {                

```

Comment:
Injektált service-nél nem kellene, hogy `using`-okat kelljen használni (amúgy hibás a brace styling), mert ugyanezt a példányt más classok is használhatják. A dependency injection scope lejártakor szerintem amúgy is dispose-olja a container, de azért nézd meg.